### ISSUE TITLE
How to create a hotkey for the plus character +?

### ISSUE THREAD

**Issue by josdejong** [JohannesKlauss/react-hotkeys-hook#222]
Thanks for sharing `useHotkeys` Johannes, it works like a charm :+1: 

Question: how to create a hotkey for the plus key `+`?

Looking at [the docs of hotkeys](https://github.com/jaywcjlove/hotkeys/#option), you have to specify a custom `splitKey`: 

```js
hotkeys('ctrl-+', { splitKey: '-' }, function(e) {
  console.log('you pressed ctrl and +');
});

hotkeys('+', { splitKey: '-' }, function(e){
  console.log('you pressed +');
})
```

As far as I can see, there is currently no way to pass options via `useHotkeys` to `hotkeys`. Any idea?

**Comment by JohannesKlauss** [comment]
@josdejong thx for reporting the issue. You are right, currently there is no way to define that. I'll work on an api asap, so you can define a split key.

**Comment by josdejong** [comment]
:+1: thanks

**Comment by josdejong** [comment]
Any news on this issue Johannes?

**Comment by JohannesKlauss** [comment]
@josdejong Yes, I am currently working on a version 2 that includes all the other requested changes and will use a new function signature to be more compliant with all the user wishes. Just bear with me :)

**Comment by josdejong** [comment]
That sounds promising 😎 , thanks for the update!

**Comment by JohannesKlauss** [comment]
I just released version 2.0.1 which include the ability to pass options and with that the `splitKey` option. Pls note that the deps and options parameter have switched places in v2

**Comment by josdejong** [comment]
Thanks Johannes that is good news, I can confirm it works like a charm :+1:
